config.yml

```diff
@@ -21,6 +21,7 @@ defaults:
     taskGroupActiveSetTableName:  TaskGroupActiveSets
     taskRequirementTableName:     TaskRequirement
     taskDependencyTableName:      TaskDependency
+    workerTypeTableName:          WorkerType
```

Comment:
I guess we could just choose not to care, rather than debate this :)

I think `TaskRequirement` and `TaskDependency` are singular because they are relations, more than they are data-tables...